corehq/apps/reports/util.py

```diff
@@ -192,7 +193,7 @@ def get(self, item, default=None):
 
 
 class SimplifiedUserInfo(
-        namedtupledict('SimplifiedUserInfo', (
+        namedtupledict(b'SimplifiedUserInfo', (
```

Comment:
I think this will fail on Python 3
```
$ python -c "from collections import namedtuple; namedtuple(b'abc', 'abc def')"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File ".../python3.5/collections/__init__.py", line 384, in namedtuple
    'identifiers: %r' % name)
ValueError: Type names and field names must be valid identifiers: "b'abc'"
```
Maybe change to `namedtupledict(str('SimplifiedUserInfo'), (`

Reply:
Does https://github.com/dimagi/commcare-hq/pull/19641/commits/4b45776b14f0fdcc525edee9d7e865ecd768680a satisfy both python versions?

Reply:
Yes... I guess you couldn't use `str('SimplifiedUserInfo')` because python-modernize doesn't like it? That's a shame.

Reply:
Indeed :(

Reply:
Just realized you can do this with ```six.binary_type(u'SimplifiedUserInfo')```

Reply:
Does that work on Python 3? I think not because it needs to be text (unicode) on Python 3.

Reply:
ah, of course